tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -353,6 +353,14 @@ batch:
         compute-type: BUILD_GENERAL1_LARGE
         image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-9x_latest
 
+    - identifier: openssh_integration
+      buildspec: ./tests/ci/codebuild/linux-x86/openssh_integration.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_LARGE
+        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:amazonlinux-2022_clang-14x_latest
```

Comment:
Since you're running the clang image can we build AWS-LC and OpenSSH with address sanitizer and make sure AWS-LC's memory management is the same as OpenSSL and nothing is leaked?

Reply:
Added ASAN to the AWS-LC build in the latest commit.